components/list-steps/list-steps-en.html

```diff
@@ -587,3 +588,56 @@ <h4>Example with heading and section of step following a step started at 9, expe
 		</li>
 	</ol>
 </section>
+<section>
+	<h2 id="limitwidth">Limited width content</h2>
+	<p>List items are normally restricted in character width when using the class <code>cnt-wdth-lmtd</code>. A list step that contain a <code>&lt;div&gt;</code>, <code>&lt;section&gt;</code> or <code>&lt;table&gt;</code> tag will not be restricted in character width, nor the components within. </p>
+	<p>However, elements normally restricted by the <code>cnt-wdth-lmtd</code> class will still be effected within those tags.</p>
+	<section class="cnt-wdth-lmtd">
+		<h3>With cnt-wdth-lmtd class</h3>
+		<ol class="lst-stps ld-zr">
+			<li>
+				<h5>Header for step 1</h5>
```

Comment:
```suggestion
				<h4>Header for step 1</45>
```

Reply:
Need to fix edit for close tag /45 to /h4